### ISSUE TITLE
Concatenate word embedding with external features

### ISSUE THREAD

**Issue by krayush07** [tflearn/tflearn#230]
Suppose I obtain an input matrix after embedding lookup which looks like:

[ [[`0.5, 0.25, 0.47, 0.86`], [`0.8. 0.12, 0.63, 0.97`], [`0.7, 0.47, 0.32, 0.01`]], ..., [[]] ] i.,e each embedding is of dim = 4 and sentence length be 3 as given in the mentioned case.

How can we append a feature vector of dim say 2, corresponding to each word in the sentence dynamically (i.e, during run time) using the placeholder? So `final input will be of dim = embedding_dim + feature_dim.`

**Comment by aymericdamien** [comment]
how is generated that feature vector? Is it define by you? So then you can simply add another 'input_data' layer to your model and concatenate it with your embedding (see tf.concat).
Then you feed the feature vector the same way that you feed your data: `model.fit([Xdata, Xfeatures], Y)`

**Comment by saja1994** [comment]
Hello. Have anyone found the answer to this question?
